nossas/bonde
# Feature import activists on crm

## How to test

1. Open `/activists` page
2. Click on `Importar CSV` button
3. Upload csv file with `first_name`, `last_name` and `email` like header
4. Should show message of success or fail

## Blocked by
- https://github.com/nossas/bonde-server/pull/387
